index_src.js

```diff
@@ -96,12 +96,15 @@ function RukusSignal(RukusApp, riot) {
         // { signal : 'name',
         //   desc : 'sig description',
         //   data : 'data accepted description',
-        //   func : function to call
+        //   func : function to call OR string name of function to call
         // }
         obj.accepts = (...signals) => {
             signals.forEach(function(sig) {
                 obj.__accepts[sig.signal] = sig;
-                obj.on(sig.signal, sig.func);
+                var func = typeof sig.func == 'function' ? sig.func : function() {
```

Comment:
Consider naming the function, ie:  function acceptsWrapper() {  ... 
Makes it easier to locate in the debugger